container/image_test.py

```diff
@@ -706,18 +706,6 @@ def test_java_runfiles_image(self):
         './app/io_bazel_rules_docker/testdata/java_runfiles_image.classpath'
       ])
 
-      self.assertConfigEqual(img, 'Entrypoint', [
-        '/usr/bin/java', '-cp',
-        ':'.join([
-          '/app/io_bazel_rules_docker/../bazel_tools/tools/java/runfiles/librunfiles.jar',
-          '/app/io_bazel_rules_docker/testdata/java_runfiles_image.binary.jar',
-          '/app/io_bazel_rules_docker/testdata/java_runfiles_image.binary',
-          '/app/io_bazel_rules_docker/testdata/foo',
-        ]),
-        'examples.images.Runfiles',
-      ])
-
-      
   def test_java_runfiles_as_lib_image(self):
```

Comment:
can we change this to also use images in tests/docker/java/BUILD, and remove those targets in testdata/BUILD?

Reply:
will be doing that in an upcoming PR, I want to migrate things progressively to start getting everything out of testdata/BUILD that is a big mess

Reply:
sg.